libs/dh-app/core/feature-shell/src/lib/shell/shell.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { WattModule } from '@energinet/watt';
+
+import { ShellComponent } from './shell.component';
+
+@NgModule({
+  declarations: [ShellComponent],
+  imports: [WattModule],
+})
+export class ShellModule {}
```

Comment:
suggestion: support router outlet for app features

```suggestion
import { NgModule } from '@angular/core';
import { RouterModule } from '@angular/router';
import { WattModule } from '@energinet/watt';

import { ShellComponent } from './shell.component';

@NgModule({
  declarations: [ShellComponent],
  imports: [RouterModule, WattModule],
})
export class ShellModule {}

```